openslides/agenda/static/templates/agenda/item-detail.html

```diff
@@ -31,26 +31,53 @@ <h3 os-perm="agenda.can_manage" translate>Comment</h3>
 </div>
 
 
-<h3 translate>List of speakers</h3>
+<h3 translate>List of speakers
```

Comment:
To use label-span-tag within of h3-tag is recommended by bootstrap, see http://getbootstrap.com/components/#example
And I like it to have the button beside the label...